src/facebook.py

```diff
@@ -40,6 +40,7 @@
 import random
 import mimetypes
 import httplib
+import requests
```

Comment:
If you're going to include requests, please adjust the setup.py to make sure it's a required package. Here's a pretty good way to make that happen: https://github.com/eventbrite/eventbrite-sdk-python/blob/master/setup.py